cmd/director.go

```diff
@@ -41,4 +41,10 @@ func init() {
 	if err != nil {
 		panic(err)
 	}
+
+	directorServeCmd.Flags().StringP("default-endpoint", "e", "", "Set whether the default endpoint should redirect to caches or origins")
+	err = viper.BindPFlag("DirectorDefaultEndpoint", directorServeCmd.Flags().Lookup("default-endpoint"))
```

Comment:
Let's use the hierarchical capabilities of viper:
```suggestion
	err = viper.BindPFlag("Director.DefaultResponse", directorServeCmd.Flags().Lookup("default-endpoint"))
```